pyanaconda/constants.py

```diff
@@ -139,10 +139,12 @@
 PASSWORD_EMPTY_ERROR = N_("The password is empty.")
 PASSWORD_CONFIRM_ERROR_GUI = N_("The passwords do not match.")
 PASSWORD_CONFIRM_ERROR_TUI = N_("The passwords you entered were different.  Please try again.")
+PASSWORD_TOO_SHORT = N_("The password is too short")
```

Comment:
This is actually correct - the message replaces error messages from `libpwquality` if the password is too short and `libpwquality` apparently does not end it's errors with a dot (even though they start with an upper case letter).

But this is not apparent at all, so I'll add a comment explaining this to next to the constant. :)